test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp

```diff
@@ -224,7 +225,7 @@ struct MultiFlagCreationTest : public BasicCommandBufferTest
         }
 
         // If we can't find multiple supported flags, still set a bitfield but
-        // expect CL_INVALID_PROPERTY to be returned on creation.
+        // expect CL_INVALID_VALUE to be returned on creation.
```

Comment:
Looking at this test again, I'm thinking that now we have specific negative tests for command-buffer creation properties it might be better to remove the negative testing component from this test and just try to create a command-buffer with as many flags as are valid. In theory that shouldn't reduce test coverage, but simplify this test.

Reply:
This makes sense to me, and I don't mind making the change in this PR if we all agree.

To be clear, what this will look like is: instead of having this special code block checking for `num_flags_set < 2`, we will just find as many flags as we can set, and try to create a command buffer with those flags.  This command buffer creation must succeed, so no `CL_INVALID_VALUE` or `CL_INVALID_PROPERTY` check will be needed.

Reply:
I've made this change and this test is much simpler now - please take a look.  Thanks!